content/en/integrations/aws-cli/index.md

```diff
@@ -117,6 +117,10 @@ $ . .venv/bin/activate
 $ pip install https://github.com/boto/botocore/archive/v2.zip https://github.com/aws/aws-cli/archive/v2.zip
 {{< / command >}}
 
+Please also note there is a known limitation for using `--no-sign-request` in
+your requests using the AWS CLI which causes localstack to not be able to route
+your request to the correct service.
```

Comment:
Can you add a small note of what the limitation is and how should the user address it (maybe by using `awslocal` CLI)